contrib/gorm.io/gorm.v1/gorm_test.go

```diff
@@ -343,3 +343,53 @@ func TestContext(t *testing.T) {
 		assert.Equal(t, testCtx.Value(key(contextKey)), ctx.Value(key(contextKey)))
 	})
 }
+
+func TestError(t *testing.T) {
```

Comment:
This test case [seems to fail](https://app.circleci.com/pipelines/github/DataDog/dd-trace-go/1459/workflows/876b18fa-c681-4425-bb5c-2d7b79e43965/jobs/9962?invite=true#step-115-1146).

<img width="680" alt="2021-06-29 Google Chrome test-contrib (9962) - DataDogdd-trace-go - 11-09-59" src="https://user-images.githubusercontent.com/15000/123770400-8ddd0e00-d8ca-11eb-99b5-1c057e85c94e.png">

Reply:
Fixed.